src/index.js

```diff
@@ -316,10 +317,22 @@ aws.config.region = "${region}";
 const lambda = new aws.Lambda();
 const functions = ${JSON.stringify(functions)};
 
+Promise.delay = function(t, val) {
```

Comment:
Extending built-in types and classes is never a good idea.

Reply:
The way it was done, without any pre-existence check, yes. I blindly C&P'd from SO and that's usually a bad idea.  However I'm game with just making these functions in the current scope or some other better plan.